### PR TITLE
ENT-5051 support revert of Federated Reporting changes

### PR DESCRIPTION
**Pull request by craigcomstock**
Will work on this one first and then fixup the master version (https://github.com/cfengine/masterfiles/pull/1572)

### CODE REVIEW

File: cfe_internal/enterprise/federation/federation.cf [hunk 1]
```diff
@@ -34,9 +39,6 @@ bundle agent config
       "data" data => readjson( $(path) );
```
[the remark this answers is not in the document]
Comment (craigcomstock) [reply cfe_internal/enterprise/federation/federation.cf]: Right. It's a bit odd in terms of wording. We have enabled and then target_state I suppose. Better wording would be fr_config_present and maybe target_state_<on|paused|off>?

File: cfe_internal/enterprise/federation/federation.cf [hunk 2]
```diff
@@ -153,7 +160,8 @@ bundle agent transport_user
           "$(ssh_known_hosts)",
           "$(ssh_config)"
         };
-      "ssh_home_cil_file" string => "$(cfengine_enterprise_federation:config.federation_dir)/cftransport_ssh_home_t.cil";
+      "ssh_home_cil_name" string => "cftransport_ssh_home_t";
```
[the remark this answers is not in the document]
Comment (craigcomstock) [reply cfe_internal/enterprise/federation/federation.cf]: We will migrate away from cil entirely after I finish work on https://github.com/cfengine/masterfiles/pull/1582

**Comment by craigcomstock** [comment]
working only on https://github.com/cfengine/masterfiles/pull/1572 so closing this one. Will cherry-pick the 3.12.x version after the master is finished.

File: cfe_internal/enterprise/federation/federation.cf (hunk 1 above)
[the remark this answers is not in the document]
Comment (nickanderson) [reply cfe_internal/enterprise/federation/federation.cf]: yes, i like those better.

**Pull request closed without merging**